deepmd/pd/model/descriptor/se_a.py

```
@@ -0,0 +1,680 @@
+
+@DescriptorBlock.register("se_e2_a")
+class DescrptBlockSeA(DescriptorBlock):
+    ndescrpt: Final[int]
+    __constants__: ClassVar[list] = ["ndescrpt"]
+
+    def __init__(
+        self,
+        rcut,
+        rcut_smth,
+        sel,
+        neuron=[25, 50, 100],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)